http-gateway/pkg/server/topic_existence_checker.go

```diff
@@ -91,16 +88,25 @@ func NewRiffTopicExistenceChecker(clientSet *versioned.Clientset, stop <-chan st
 	return &riffTopicExistenceChecker{topicInformer: topicInformer, mutex: mutex, knownTopics: knownTopics}
 }
 
-func (tec *alwaysTrueTopicExistenceChecker) TopicExists(namespace string, topicName string) bool {
+func topicName(key string) string {
```

Comment:
Turns out there's a provided method for this: `cache.SplitMetaNamespaceKey`. Same package as the `MetaNamespaceKeyFunc` used in the `AddFunc` informer callback.